agent/agent-profiler/agent-alerting/build.gradle.kts

```diff
@@ -6,6 +6,9 @@ dependencies {
   implementation(project(":agent:agent-profiler:agent-alerting-api"))
   implementation("org.slf4j:slf4j-api")
 
+  implementation("com.google.guava:guava")
```

Comment:
we don't already have guava in the javaagent, would love not to pull in the extra 2.5 mb if possible